plugins/octant/cmd/antrea-octant-plugin/main.go

```diff
@@ -42,28 +38,36 @@ const (
 	overviewTitle   = "Overview"
 	controllerTitle = "Antrea Controller Info"
 	agentTitle      = "Antrea Agent Info"
-	versionCol      = "Version"
-	podCol          = "Pod"
-	nodeCol         = "Node"
-	serviceCol      = "Service"
-	crdCol          = "Monitoring CRD"
-	subnetCol       = "NodeSubnet"
-	bridgeCol       = "OVS Bridge"
-	podNumCol       = "Local Pod Num"
-	heartbeatCol    = "Last Heartbeat Time"
+	traceflowTitle  = "Trace Flow"
 )
 
-func main() {
-	// Remove the prefix from the go logger since Octant will print logs with timestamps.
-	log.SetPrefix("")
+type localPlugin struct {
```

Comment:
I prefer to change the name to "octantPlugin". What do you think?

Reply:
I agree with you.